deepset_cloud_sdk/workflows/async_client/files.py

```diff
@@ -0,0 +1,83 @@
+"""This module contains async functions for uploading files and folders to the Deepset Cloud."""
+from pathlib import Path
+from typing import List, Optional
+
+from deepset_cloud_sdk.api.config import (
+    API_KEY,
+    API_URL,
+    DEFAULT_WORKSPACE_NAME,
+    CommonConfig,
+)
+from deepset_cloud_sdk.service.files_service import DeepsetCloudFiles, FilesService
+
+
+async def upload_file_paths(
+    file_paths: List[Path],
+    api_key: Optional[str] = None,
+    api_url: Optional[str] = None,
+    workspace_name: str = DEFAULT_WORKSPACE_NAME,
+    blocking: bool = True,
```

Comment:
What do you think if we rename `blocking` to something more directly associated with the uploads, e.g. `wait_until_completion`

Reply:
sure :) will rename